triggers/tests/src/test/scala/com/digitalasset/daml/lf/engine/trigger/test/LoadTesting.scala

```diff
@@ -22,6 +23,7 @@ import com.daml.util.Ctx
 import org.scalatest.{Inside, TryValues}
 import org.scalatest.matchers.should.Matchers
 import org.scalatest.wordspec.AsyncWordSpec
+import scalaz.syntax.tag._
```

Comment:
Changes in this file all relate to using `Party` rather than `String` as a method argument